user/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db               import models
+
+from .validators import (
+    phone_number_validation_function,
+    password_validation_function
+)
+
+class User(models.Model): 
+    name = models.CharField(
+        max_length = 16
+        )
+    birth = models.DateField(
+        max_length = 16
+        )
+    identifier = models.CharField(
+        max_length = 50,
+        unique     = True
+        )
+    password = models.CharField(
+        max_length = 50,
+        validators = [password_validation_function]
+        )
+    gender = models.CharField(
+        max_length = 2
+        )
+    phone_number = models.CharField(
+        validators = [phone_number_validation_function],
+        max_length = 17,
+        unique     = True
+        )
+    email = models.EmailField(
+        max_length = 254,
+        unique     = True
+        )
+
+    class Meta: 
+        db_table = "users"
+
+class User_information(models.Model): 
+    endrolled_giftcard = models.PositiveIntegerField()
+    coupon             = models.PositiveIntegerField()
+    inquiry_history    = models.PositiveIntegerField()
+    user               = models.OneToOneField(User, on_delete = models.CASCADE)
+
+    class Meta: 
+        db_table = "user_informations"
```

Comment:
```suggestion
        db_table = "user_information"
```
information은 그 자체로 복수형 의미를 줄 수 있습니다.